### ISSUE TITLE
which function returns the mean and variance of the latent function (f) at the points Xnew.[Docs]

### ISSUE THREAD

**Issue by irum** [cornellius-gp/gpytorch#1235]
Could you please explain which function is used in Gpytorch to get the values of the latent function?

In [example](https://docs.gpytorch.ai/en/v1.1.1/examples/06_PyTorch_NN_Integration_DKL/Deep_Kernel_Learning_DenseNet_CIFAR_Tutorial.html)    in test section   `output = model(data)` gives values for the dimension like (1,4096) if last layer'dimension = 4096 but the latent function 's values should be like (1,10) as I have 10 classes to be predicted.

**Comment by jacobrgardner** [comment]
I think you are assuming you have to have the same number of latent GPs as classes, which is not true. The softmax likelihood mixes the output GPs with a matrix W. If you have H output GPs and C classes, W is H x C so that `fW` is a (random) set of `C` logits. If you have W GPs, you have W latent function outputs. 

Since `f` is MultivariateNormal and `W` is a constant linear transformation, you could easily manually construct a distribution for `fW` if you really wanted to I guess.
